src/traces/contourcarpet/index.js

```diff
@@ -15,7 +15,7 @@ ContourCarpet.supplyDefaults = require('./defaults');
 ContourCarpet.colorbar = require('../contour/colorbar');
 ContourCarpet.calc = require('./calc');
 ContourCarpet.plot = require('./plot');
-ContourCarpet.style = require('./style');
```

Comment:
> do you think it's sufficient to rely

We have plenty of carpet mocks, maybe you could update one of them with `contours.coloring='lines'` ?